### DISCUSSION TITLE
Norwegian Service Provider - SIM

### DISCUSSION THREAD

**Discussion by 21espeland** [mampfes/hacs_waste_collection_schedule#3347] [Q&A]
Has any one attempted to add the Norwegian SIM to the integration. 
https://sim.as/

**Comment by 5ila5** [comment]
please open a source request here: https://github.com/mampfes/hacs_waste_collection_schedule/issues/new?assignees=&labels=source+request&projects=&template=SOURCE-REQUEST.yml&title=%5BSource+Request%5D%3A+
